scripts/load-test/collect-pod-logs/README.md

```diff
@@ -0,0 +1,36 @@
+# JIRA task - https://issues.redhat.com/browse/STONE-831 
+
+# The bash script is collect-openshift-pod-logs.sh
+
+
+# The Jira task description:
+
+Description
+===========
+Goal is to be able to look at was happening in the cluster pods logs before the cluster dies.
+
+Input of the tool would be:
+it would be running on the host logged to the StoneSoup cluster
+it will be provided a list of namespaces
+The tool would check all the pods in all the namespaces and would be following (tailing) logs to files.
+
+
```

Comment:
Hi @pmacik 
Thanks for your comment

I wasn't totally sure, so I asked chat GPT :
While it's helpful to have a script description in the JIRA ticket, it's generally best to include context in multiple locations for better visibility and accessibility. Here are a few recommendations:

1. JIRA ticket: Include a brief description of the script, its purpose, and any relevant dependencies or stakeholders. This helps team members understand the scope and importance of the task.

2. Source code comments: Add detailed comments within the script itself, explaining the purpose of specific sections, any assumptions made, and any limitations or potential improvements.

3. README or documentation: **If the script is part of a larger project or repository, include a detailed description and context in the README or associated documentation**. This will help other team members or future contributors understand how the script fits within the project and how to use or modify it.

4. Code review: If your team follows a code review process, the script's purpose and context should be explained during the review to ensure that the reviewer understands the intent and can provide relevant feedback.

By providing context in multiple locations, you make it easier for team members to understand the purpose and importance of the script, which can lead to better collaboration, maintenance, and overall project success.

In paragraph 3 it states the the context is important

Reply:
Hi @naftalysh, 

I agree with @pmacik. Let's have in readme just short description, what the script can be used for and what it does(with a link to JIRA) and instructions how to use it. Description here should be in JIRA - as it looks like more requirements and in PR description(as it is in paragraph 4).

For paragraph 3: context is important, but all the information are in JIRA(and linked issues), why duplicate the same information - in case you need to change the script, you need to verify more locations(and people are forgetting this step).

Josef

Reply:
I removed that part, soon commiting it..